src/vizier/services/shared/agentpb/agent.proto

```diff
@@ -30,12 +30,17 @@ message AgentCapabilities {
   bool collects_data = 1;
 }
 
+message AgentParameters {
+  int32 profiler_stack_trace_sample_period_ms = 1;
```

Comment:
Nit: We probably want `[ (gogoproto.customname) = "ProfilerStackTraceSamplePeriodMS" ];`